templates/solaris_install.sh.erb

```diff
@@ -1,11 +1,13 @@
 function start_service() {
   service="${1:?}"
-  /opt/puppetlabs/bin/puppet resource service "${service:?}" ensure=running enable=true
```

Comment:
Why remove `enable=true`?

Reply:
@MikaelSmith sorry for the delay I was on PTO. Can you check again please?